packages/theming/android-theme/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@fluentui-react-native/android-theme",
+  "version": "0.1.0",
+  "description": "A FluentUI React Native theme that pulls constants from FluentUI Android",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/just"
+  },
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "bundle": "fluentui-scripts bundle",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "code-style": "fluentui-scripts code-style",
+    "depcheck": "fluentui-scripts depcheck",
+    "lint": "fluentui-scripts eslint",
+    "start": "fluentui-scripts dev",
+    "start-test": "fluentui-scripts jest-watch",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@fluentui-react-native/theme-types": ">=0.5.0 <1.0.0",
+    "@fluentui-react-native/theme": ">=0.4.1 <1.0.0"
+  },
+  "devDependencies": {
+    "@uifabricshared/build-native": "^0.1.1",
+    "@uifabricshared/eslint-config-rules": "^0.1.1"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1",
+    "react-native": "^0.62.2"
```

Comment:
We recently updated the repo to use React Native 0.63 as the minimum, would you be able to update this here too?
https://github.com/microsoft/fluentui-react-native/pull/631

Reply:
updated to 0.63.4 =)